docs/clients_tfs.md

```diff
@@ -0,0 +1,237 @@
+
+        import grpc
+        from tensorflow_serving.apis import prediction_service_pb2_grpc, predict_pb2
+        from tensorflow import make_tensor_proto, make_ndarray
+
+        channel = grpc.insecure_channel("10.20.30.40:9000")
+        prediction_service_stub = prediction_service_pb2_grpc.PredictionServiceStub(channel)
+
+        with open("img.jpeg", "rb") as f:
+            data = f.read()
+        predict_request = predict_pb2.PredictRequest()
+        predict_request.model_spec.name = "my_model"
+        predict_request.inputs["input_name"].CopyFrom(make_tensor_proto(data))
+        predict_response = prediction_service_stub.Predict(predict_request, 10.0)
+        results = make_ndarray(predict_response.outputs["output_name"])
+
+@endsphinxdirective
+
+### Request Prediction on a Numpy Array
+
+@sphinxdirective
+
+.. tab:: ovmsclient
+
+    .. code-block:: python
+
+        import numpy as np
+        from ovmsclient import make_grpc_client
+
+        client = make_grpc_client("10.20.30.40:9000")
+        data = np.array([1.0, 2.0, ..., 1000.0])
+        inputs = {"input_name": data}
+        results = client.predict(inputs=inputs, model_name="my_model")
+
+
+.. tab:: tensorflow-serving-api  
+
+    .. code-block:: python
+
+        import grpc
+        from tensorflow_serving.apis import prediction_service_pb2_grpc, predict_pb2
+        from tensorflow import make_tensor_proto
+
+        channel = grpc.insecure_channel("10.20.30.40:9000")
+        prediction_service_stub = prediction_service_pb2_grpc.PredictionServiceStub(channel)
+
+        data = np.array([1.0, 2.0, ..., 1000.0])
+        predict_request = predict_pb2.PredictRequest()
+        predict_request.model_spec.name = "my_model"
+        predict_request.inputs["input_name"].CopyFrom(make_tensor_proto(data))
+        predict_response = prediction_service_stub.Predict(predict_request, 10.0)
+        results = make_ndarray(predict_response.outputs["output_name"])
+
+@endsphinxdirective
+
+For complete usage examples see [ovmsclient samples](https://github.com/openvinotoolkit/model_server/tree/releases/2022/1/client/python/ovmsclient/samples).
+
+## C++ Client
+
+@sphinxdirective
+.. raw:: html
+
+    <div id="switcher-cpp" class="switcher-anchor">C++</div>
+@endsphinxdirective
+
+Creating a client application in C++ follows the same principles as Python, but using C++ adds some complexity. There is no package or library available with convenient functions for interacting with OpenVINO Model Server.
+
+To successfully set up communication with the model server you need to implement the logic to communicate with endpoints specified in the [API](api_reference_guide.md). For gRPC, download and compile protos, then link and use them in your application according to the [gRPC API specification](model_server_grpc_api_tfs.md). For REST, prepare your data and pack it into the appropriate JSON structure according to the [REST API specification](model_server_rest_api_tfs.md).
```

Comment:
We could point the user here to our c++ benchmark app. While it's purpose is not to show exactly how to prepare c++ proto's interface it may be a good start. What do you think?